app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb

```diff
@@ -1,4 +1,8 @@
 module ManageIQ::Providers
   class Lenovo::PhysicalInfraManager::PhysicalServer < ::PhysicalServer
   end
+
+  def self.display_name(number = 1)
```

Comment:
This should be defined inside the PhysicalServer Class rather then inside the module